release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -488,6 +488,11 @@ The `openshiftVersion` parameter of the `AgentServiceConfig` custom resource (CR
 
 For more information see, xref:../scalability_and_performance/routing-optimization.adoc#configuring-ingress-controller-router-liveness-readiness-and-startup-probes[Configuring Ingress Controller (router) Liveness, Readiness, and Startup probes]
 
+[id=ocp-4-11-nto-offlined-cpu]
+==== New offlined CPU capability
```

Comment:
Without a direct reference to the parameter, "offlined" is either jargon or an application of a complex verb tense as an adjective. Please be more descriptive. Maybe:
```suggestion
==== New power reduction capability
```